packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -189,11 +189,18 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                 this.setState({
                     outputStatus: treeResponse.status,
                     timegraphTree: treeResponse.model.entries,
-                    columns,
+                    columns
```

Comment:
Removed this comma because I assumed it was a typo and things still worked when I did my usual tests.